configureWebpack.js

```diff
@@ -601,6 +607,11 @@ async function configureWebpack(env) {
                             options: {
                                 name: 'static/media/[name].[hash:8].[ext]'
                             }
+                        },
+                        {
+                            test: /\.js$/,
+                            enforce: "pre",
+                            use: ["source-map-loader"],
```

Comment:
How will we hit this new rule, given the `oneOf` block it's nested underneath? I would expect any js file to be captured and processed by the second rule in the array, and would not think that anything would hit this new rule.

Did you see any effect from it?